release_notes/Service_Fabric_ReleaseNotes_90.md

```diff
@@ -0,0 +1,153 @@
+| 11. | **Feature** | Introduced a property in Service Fabric runtime that can be set via SFRP as the ARM resource id | **Brief Description**: To better support Service Fabric Resource Provider (SFRP) & Service Fabric managed clusters (SFMC) scenarios, SF runtime needs to allow support for additional metadata for ARM deployed resources.<br>**Impact**: Metadata added for application type are application type provision timestamp and arm resource id. Metadata added for application, service, and cluster are ARM resource id.<br>Existing API's have been updated to read this metadata and support for Powershell, REST, and FabricClient have been added.<br>The runtime created a new API for Service Fabric Resource Provider (SFRP) for updating the metadata. Currently no public support is provided for Powershell since this is only available only for Service Fabric Resource Provider (SFRP) & Service Fabric managed clusters (SFMC) use<br>**Documentation Reference**: Read more about it at - <ul><li>ApplicationTypeMetadata</li><li>ApplicationMetadata</li><li>ServiceMetadata</li></ul> |
+| 12. | **Feature** | Exposed application type provision timestamp | **Brief Description**: Added application type provision timestamp that is exposed via existing application type API's.<br>**Impact**: To better support Service Fabric Resource Provider (SFRP) and Service Fabric managed clusters (SFMC) scenarios, SF runtime needs to allow support for additional metadata for ARM deployed resources.<br>**Documentation Reference**: Read more about it at - <ul><li>ApplicationTypeMetadata</li></ul> |
+| 13. | **Feature** | Exposed API for reporting MoveCost | **Brief Description**: Exposed API for reporting MoveCost on behalf of other service replicas and partitions.<br>**Impact**: MoveCost could be changed with action outside of partition. If clients don't own service logic, they could use only static load.<br>**Solution**: New API allows reporting MoveCost from outside of the service, which could be beneficial for clients that don't have service logic.<br>**Documentation Reference**: Read more about it at - <ul><li>[Reporting move cost for a partition](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-resource-manager-movement-cost#reporting-move-cost-for-a-partition)</li></ul>**Workaround**: Replicas could report MoveCost values for themselves or clients could specify static MoveCost value during service creation. |
```

Comment:
What can customers do with this move cost value? Can they prevent certain replicas and partitions from being moved within the cluster?